tests/snapshots/files__if_in_loop-optimize.snap

```diff
@@ -7,28 +7,24 @@ expression: visualization.result
   c1_: int = const 0;
   c2_: int = const 1;
   c3_: int = const 10;
-  v4_: bool = lt v0 c2_;
-  v5_: int = select v4_ c2_ c1_;
-  v6_: int = id c1_;
-  v7_: int = id c2_;
-  v8_: int = id v0;
-  v9_: int = id c1_;
-  v10_: int = id c3_;
-  v11_: bool = id v4_;
-  v12_: int = id v5_;
-.b13_:
-  print v12_;
+  v4_: int = id c1_;
+  v5_: int = id c2_;
+  v6_: int = id v0;
+  v7_: int = id c1_;
+  v8_: int = id c3_;
+.b9_:
+  v10_: bool = lt v6_ v5_;
+  v11_: int = select v10_ v5_ v7_;
```

Comment:
The code motion is undone by this PR, because it thinks this uses more registers.